README.md

```diff
@@ -1,24 +1,69 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Re-Former
 
-Things you may want to cover:
+-Models, Views and Controllers (MVC)
+-Made as a practice for form validations in rails.
+-Form allows to create new users and edit them.
+-Field information must fulfill validations.
 
-* Ruby version
+## Built With
+- Ruby
+- Ruby on rails
+- SQLite3
 
-* System dependencies
+## Pre-requisites
+- Ruby 2.7.0
+- Rails gem
+- Bundle
 
-* Configuration
+### Setup
 
-* Database creation
+-Open your terminal and go to the directory where you want to clone the repo.
 
-* Database initialization
+-Clone the repository to your local machine. Type $ git clone `git@github.com:https://github.com/NataJenkins/re-former-`
+-Go to the Micro-reddit directory. Type $ cd Micro-reddit
+-Install the necessary gems to run the project. Type $ bundle install
+-Also please run this command `rails db:migrate` to run all the migration for the database and have all tables updated and read to use
+-Now your environment is ready to run the project. Type rails server
```

Comment:
**_[Optional] Try to format the setup instructions. I think you need to put space between the hyphen(-) at the beginning of each line and the sentences_**